python/paddle/nn/functional/loss.py

```diff
@@ -4280,3 +4280,151 @@ def gaussian_nll_loss(
         return paddle.sum(loss, name=name)
     elif reduction == 'none':
         return loss
+
+
+def adaptive_log_softmax_with_loss(
+    input, label, head_weight, tail_weights, cutoffs, head_bias=None, name=None
+):
+    r"""Compute adaptive logsoftmax result and negative log likelihood between ``input`` and ``label``.
+    Parameter ``head``, ``tail_weights``, ``cutoffs`` are inner members of AdaptiveLogSoftmaxWithLoss
+    Please refer to :ref:`_cn_api_paddle_nn_AdaptiveLogSoftmaxWithLoss`.
+
+    Args:
+        input (Tensor): Input tensor, the data type should be float32 or float64.
+        label (Tensor): Label tensor, the data type should be float32 or float64.
+        head_weight (Tensor): weight tensor for linear computation, the data type should be float32 or float64.
+        tail_weights (Tensor): weight tensor for linear computation, the data type should be float32 or float64.
```

Comment:
Because `head_weight` and `tail_weights` are not default parameters, it is necessary to clearly inform the user of the shape requirements for `head_weight` and `tail_weights`

Reply:
> Because `head_weight` and `tail_weights` are not default parameters, it is necessary to clearly inform the user of the shape requirements for `head_weight` and `tail_weights`

Okay, I'll make it up soon

Reply:
> Because `head_weight` and `tail_weights` are not default parameters, it is necessary to clearly inform the user of the shape requirements for `head_weight` and `tail_weights`

It‘s been updated